### PR TITLE
GOVUKAPP-472 Fonts

### PR DESCRIPTION
**Pull request by batesyuk**

### CODE REVIEW

File: Production/govuk_ios/ViewControllers/TabBarController.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func registerNotifications() {
+        notificationCenter.addObserver(
+            self,
+            selector: #selector(updateAppearance),
+            name: UIAccessibility.boldTextStatusDidChangeNotification,
+            object: nil
+        )
```
Comment (robinphillips) [inline Production/govuk_ios/ViewControllers/TabBarController.swift:35]: I believe you should remove observers whenever you add them to a class in a `deinit` block.

Comment (batesyuk) [reply Production/govuk_ios/ViewControllers/TabBarController.swift:35]: You don't _need_ to, but I am happy to do it if we deem it better practice to do so.
https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver#discussion

Comment (batesyuk) [reply Production/govuk_ios/ViewControllers/TabBarController.swift:35]: If we go that route, might be better to use a block, instead of selector

**Review by raphaelolu — APPROVED** [review]
This looks good to me

**Pull request closed without merging**